pbjam/session.py

```diff
@@ -298,7 +300,7 @@ def _format_col(vardf, col, key):
                              np.array([_arr_to_lk(x, y, vardf.loc[i, 'ID'], key)]))
         vardf[key] = temp
     else:
-        print('Unhandled exception')
+        logger.critical('Unhandled exception.')
```

Comment:
This is the best message every. I should have changed it long time ago.

Perhaps Unhandled exception when formatting input columns or something like that.

Now that it goes in the logger it's perhaps not that important though.

Reply:
That is true, the extra detail may still be useful in the logger. Alternatively we could raise an Error, especially if the unhandled exception causes issues later one?

Reply:
Yeah it's probably best to raise an error.

This may actually just be a leftover from before I knew how to raise errors properly.